.buildkite/pipeline.yml

```diff
@@ -50,22 +50,27 @@ steps:
     - nix-build ci.nix -A tzbtc.components.tests.tzbtc-test
     - ./result/bin/tzbtc-test --cleveland-mode=disable-network
 
-  - label: test-local-chain-011
+  - label: test-local-chain-012
     if: *not_scheduled_autodoc
     depends_on: build
     env:
-      TASTY_CLEVELAND_NODE_ENDPOINT: "http://localhost:8734"
+      TASTY_CLEVELAND_NODE_ENDPOINT: "http://localhost:8733"
     commands: &run-nettest
+    - export PATH=$(nix-build ci.nix -A tzbtc.components.exes.tzbtc-client)/bin:$PATH
     - nix-build ci.nix -A tzbtc.components.tests.tzbtc-test
-    - export TASTY_CLEVELAND_DATA_DIR="$(mktemp -d --tmpdir="$$PWD")"
-    - nix run -f ci.nix tezos-client tzbtc.components.exes.tzbtc-client -c ./result/bin/tzbtc-test
-      --cleveland-mode=only-network
-
-  - label: scheduled hangzhounet test
+    # Note that 'refill-balance' below is the initial 'TASTY_CLEVELAND_MONEYBAG_SECRET_KEY' balance
+    # which may need to be adjusted in case of insufficient balance errors
+    - $(nix-build ci.nix -A run-chain-tests
+        --argstr refill-balance 3500
```

Comment:
Damn, it looks like ~300 XTZs weren't used by the tests, but the rest did.

I suppose that this could be reduced to to 3300 and still give us more than enough margin.